frontend/src/App.css

```diff
@@ -36,3 +36,23 @@
     transform: rotate(360deg);
   }
 }
+
+.Games-grid {
```

Comment:
CSS naming conventions: didžiųjų raidžių nenaudojam

Reply:
lyginaus pagal .App, .App-header ir panasiai, bet okey. Cia is react'o template speju, gal ir tuos pakeist i lowercase?

Reply:
Jo reiktų